code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -214,45 +228,49 @@
+
 	var/gas_power = 0
 	for (var/id in cached_gases)
-		gas_power += cached_gases[id][GAS_META][META_GAS_FUSION_POWER]*cached_gases[id][MOLES]
-	var/plasma_fused = 0
-	var/power_ratio = min(gas_power/mediation,100)//100 is a lot, we really don't want to go over this.
-	if (power_ratio > 10) //Super-fusion. Fuses everything into one big atom which then turns to tritium instantly. Very dangerous, but super cool.
-		var/gases_fused = air.total_moles()
-		reaction_energy += gases_fused*PLASMA_BINDING_ENERGY*(gas_power/(mediation*100))
+		gas_power += reaction_efficiency * (cached_gases[id][GAS_META][META_GAS_FUSION_POWER]*cached_gases[id][MOLES])
+
+	var/power_ratio = gas_power/mediation
+
+	if (power_ratio > FUSION_HIGH_TIER) //Super-fusion. Fuses everything into one big atom which then turns to tritium instantly. Very dangerous, but super cool.
+		reaction_energy += gases_fused*FUSION_RELEASE_ENERGY*(power_ratio / FUSION_HIGH_REACTION_ENERGY_FACTOR)
+		for (var/id in cached_gases)
+			cached_gases[id][MOLES] = 0
+		cached_gases[/datum/gas/tritium][MOLES] += gases_fused * FUSION_GAS_CREATION_FACTOR_HIGH //25% of the gas is converted to energy, 75% to tritium
+		if (location && prob(power_ratio)) //You don't really want this to happen
+			radiation_pulse(location, reaction_energy / FUSION_HIGH_RADIOACTIVITY_FACTOR)
+			explosion(location,0,0,3,power_ratio * 0.5,TRUE,TRUE)//A tiny explosion with a large shockwave. People will know you're doing fusion.
+
+	else if (power_ratio > FUSION_MID_TIER) //Mediation is overpowered, fusion reaction starts to break down.
+		reaction_energy += cached_gases[/datum/gas/plasma][MOLES]*FUSION_RELEASE_ENERGY
 		for (var/id in cached_gases)
 			cached_gases[id][MOLES] = 0
-		air.assert_gas(/datum/gas/tritium)
-		cached_gases[/datum/gas/tritium][MOLES] += gases_fused
-		if (prob(power_ratio)) //You really don't want this to happen
-			empulse(location, power_ratio*0.5, power_ratio)
-			radiation_pulse(location, power_ratio*2000)
-			explosion(location,0,1,power_ratio*0.5,power_ratio,TRUE,TRUE)//Bypasses cap. Doesn't blow large hole in station, but produces moderate devestation for long ranges. Be careful with this.
-
-	else if (power_ratio > 1) //Mediation is overpowered, fusion reaction starts to break down.
-		plasma_fused = cached_gases[/datum/gas/plasma][MOLES]
-		reaction_energy += plasma_fused*PLASMA_BINDING_ENERGY
-		cached_gases[/datum/gas/plasma][MOLES] -= plasma_fused
-		cached_gases[/datum/gas/carbon_dioxide][MOLES] = 0
 		air.assert_gases(/datum/gas/bz,/datum/gas/nitrous_oxide)
-		cached_gases[/datum/gas/bz][MOLES] += gas_power*0.05
-		cached_gases[/datum/gas/nitrous_oxide][MOLES] += gas_power*0.05
-		if (location)
-			empulse(location, mediation*0.002, mediation*0.004)
-			radiation_pulse(location, power_ratio*(reaction_energy)/(0.3*PLASMA_BINDING_ENERGY))
+		cached_gases[/datum/gas/bz][MOLES] += gases_fused * FUSION_GAS_CREATION_FACTOR_MID //10% of the gas is converted to energy, 90% to BZ and N2O
```

Comment:
this might be too much BZ for too little effort?

Reply:
I would say 75% N2O 25% BZ. But there is no such thing as "too much BZ for too little effort" since if you wanted to do BZ you could do the other reaction that is easier than fusion